src/main/scala/scalismo/registration/TransformationSpace.scala

```diff
@@ -796,18 +796,20 @@ case class AnisotropicScalingSpace[D: NDSpace]() extends TransformationSpace[D]
  * The order of the rigid transform in this case is also fixed : first rotate then translate.
  *
  */
-trait AnisotropicSimilarityTransformation[D] extends CompositeTransformation[D] with CanInvert[D]
+trait AnisotropicSimilarityTransformation[D] extends CompositeTransformation[D] with CanInvert[D] {
+  override def inverse : AnisotropicSimilarityTransformation[D]
+}
 
 private class RigidTransformationThenAnisotropicScaling[D: NDSpace](anisotropicScaling: AnisotropicScalingTransformation[D], rigidTransform: RigidTransformation[D])
     extends CompositeTransformation[D](anisotropicScaling, rigidTransform) with AnisotropicSimilarityTransformation[D] {
 
-  def inverse: AnisotropicScalingThenRigidTransformation[D] = new AnisotropicScalingThenRigidTransformation[D](rigidTransform.inverse, anisotropicScaling.inverse)
+  override def inverse: AnisotropicScalingThenRigidTransformation[D] = new AnisotropicScalingThenRigidTransformation[D](rigidTransform.inverse, anisotropicScaling.inverse)
```

Comment:
This might be dangerous, as we do not restrict AnisotropicScalingTransformations to be strictly positive. But let's discuss that in another issue.